packages/styles/src/components/blockquote.scss

```diff
@@ -0,0 +1,34 @@
+@use '../tokens/components';
+@use '../functions/tokens';
+@use './../placeholders/blockquote' as blockquote-ph;
+
+tokens.$default-map: components.$post-quote;
+
+.blockquote {
+  font-size: tokens.get('post-quote-font-size');
+  font-weight: tokens.get('post-quote-font-weight');
+  margin: tokens.get('post-quote-margin-block') tokens.get('post-quote-margin-inline');
+  padding: 0 tokens.get('post-quote-padding-inline-end') 0
+    tokens.get('post-quote-padding-inline-start');
+  border-inline-start: tokens.get('post-quote-border-width-left') solid
+    tokens.get('post-quote-border-left-color');
+}
+
+figure:has(.blockquote) {
+  margin: tokens.get('post-quote-margin-block') tokens.get('post-quote-margin-inline');
+  padding: 0 tokens.get('post-quote-padding-inline-end') 0
+    tokens.get('post-quote-padding-inline-start');
+  border-inline-start: tokens.get('post-quote-border-width-left') solid
+    tokens.get('post-quote-border-left-color');
+
+  .blockquote {
+    margin: 0;
+    padding: 0;
+    border: none;
+  }
+}
+
+.blockquote-footer {
+  @extend %body;
```

Comment:
This is not necessary since the figcaption inherits its font styles from the body anyways:
```suggestion
```